Makd.mak

```diff
@@ -479,7 +479,8 @@ fasttest := fastunittest $(fasttest)
 test := unittest $(test)
 
 # Files to be tested in unittests, the user could potentially add more
-UNITTEST_FILES += $(call find_files,.d,,$C/$(SRC),$(TEST_FILTER_OUT))
+UNITTEST_FILES += $(call find_files,.d,,$C/$(SRC),$(TEST_FILTER_OUT)) \
+		$(call find_files,.d,,$C/test,$(TEST_FILTER_OUT))
```

Comment:
This results in stuff like this in allunittests.d
```D
import .docker.ocean.test.unixlistener.main.d;
```
Maybe has something to do how `find_files` processes qualified paths?